ember-flight-icons/tests/dummy/app/templates/engineering.hbs

```diff
@@ -2,82 +2,139 @@
-  </ol>
+    <ol>
+      <li class="ds-li"><kbd class="ds-kbd">fill</kbd>
+        attribute: set to currentColor</li>
+      <li class="ds-li"><kbd class="ds-kbd">id</kbd>
+        attribute: a unique, automatically generated id</li>
+      <li class="ds-li"><kbd class="ds-kbd">aria-hidden</kbd>
+        attribute: set to true</li>
+      <li class="ds-li"><kbd class="ds-kbd">height</kbd>
+        and
+        <kbd class="ds-kbd">width</kbd>: default size of 16x16 (px)</li>
+      <li class="ds-li">(CSS)<kbd class="ds-kbd">class</kbd>: flight-icon, flight-icon-NAME, flight-icon-display-inline</li>
+      <li class="ds-li">CSS display: set to
+        <kbd class="ds-kbd">display:inline-block</kbd></li>
+      <li class="ds-li"><kbd class="ds-kbd">data-test-icon</kbd>
+        attribute: for the author's testing convenience; set to the value of the
+        <kbd class="ds-kbd">@name</kbd>
+        property.</li>
+    </ol>
   </p>
 
-  <p>This makes the base, required invocation quite terse- <kbd class="ds-kbd">@name</kbd> is the only property that requires specification.
-  So this invocation:
-  <CodeBlock @language="markup" id="component-invocation-base" @code="&lt;FlightIcon @name=&quot;alert-circle&quot; />" />
+  <p>This makes the base, required invocation quite terse-
+    <kbd class="ds-kbd">@name</kbd>
+    is the only property that requires specification. So this invocation:
+    {{! prettier-ignore-start }}
+    <CodeBlock
+      @language="markup"
+      id="component-invocation-base"
+      @code="&lt;FlightIcon @name=&quot;alert-circle&quot; />"
+    />
+    {{! prettier-ignore-end }}
   </p>
 
   <p>
-  Renders to this (where the ID will be unique each time):
-<CodeBlock @language="markup" id="component-invocation-base-render"
-@code="&lt;svg
- aria-hidden=&quot;true&quot;
- class=&quot;flight-icon icon-alert-circle display-inline&quot;
- data-test-icon=&quot;alert-circle&quot;
- fill=&quot;currentColor&quot;
- id=&quot;icon-ember115&quot;
- width=&quot;16&quot;
- height=&quot;16&quot;
- viewBox=&quot;0 0 16 16&quot;
- xmlns=&quot;http://www.w3.org/2000/svg&quot;>
+    Renders to this (where the ID will be unique each time):
+    {{! prettier-ignore-start }}
+    <CodeBlock
+      @language="markup"
+      id="component-invocation-base-render"
+      @code="&lt;svg
+  aria-hidden=&quot;true&quot;
+  class=&quot;flight-icon icon-alert-circle display-inline&quot;
+  data-test-icon=&quot;alert-circle&quot;
+  fill=&quot;currentColor&quot;
+  id=&quot;icon-ember115&quot;
+  width=&quot;16&quot;
+  height=&quot;16&quot;
+  viewBox=&quot;0 0 16 16&quot;
+  xmlns=&quot;http://www.w3.org/2000/svg&quot;>
   &lt;use href=&quot;/@hashicorp/ember-flight-icons/icons/sprite.svg#alert-circle-16&quot;>&lt;/use>
-&lt;/svg>" />
+&lt;/svg>"
+    />
+    {{! prettier-ignore-end }}
```

Comment:
Added prettier ignore's to every CodeBlock. FWIW CodeBlock has kinda unusual indentation as it is... L126 has to be "un-indented" to display properly on the page. 

I think this is an improvement to what we have and FWIW `boundary-ui` uses `ember-template-lint-plugin-prettier` and `cloud-ui` plans to

Can do a pre-commit hook to `yarn run lint:fix` maybe with https://pre-commit.ci/ to make it a better DX

But happy to close if y'all still disagree

Reply:
talked with @didoo and @MelSumner 

let's use a pre-push hook instead, that will lint on push

melanie shared https://github.com/okonet/lint-staged which i will look into!